src/storage/txn/commands/mod.rs

```diff
@@ -925,11 +935,31 @@ pub mod test_util {
             extra_op: ExtraOp::Noop,
             statistics,
             async_apply_prewrite: false,
+            raw_ext: None,
         };
 
         let ret = cmd.cmd.process_write(snap, context)?;
         let ctx = Context::default();
         engine.write(&ctx, ret.to_be_write).unwrap();
         Ok(())
     }
+
+    pub async fn mock_raw_ext(
+        api_ver: ApiVersion,
+        concurrency_manager: ConcurrencyManager,
+    ) -> Option<RawExt> {
+        if api_ver == ApiVersion::V2 {
+            let raw_key = vec![api_version::api_v2::RAW_KEY_PREFIX];
```

Comment:
Suggest to reuse `get_raw_key_guard`. Then we can also verify the `get_raw_key_guard` here.

Reply:
yesh, I reused it for compare_and_swap test case and atomic_store test case, thinks!